### ISSUE TITLE
debug/ developer tooling accomodations

### ISSUE THREAD

**Issue by MACKAT05** [Snowflake-Labs/schemachange#157]
Developer tooling.
I wrote a simple PowerShell script to run the render command over all the files and redirect them into a subfolder that is listed in the .gitignore
Can we add '/*' and '*/' like so?:
![image](https://user-images.githubusercontent.com/12175865/219823310-acee68a2-948a-4c6d-98f7-a640aeebb2e1.png)

This would allow the us Vs code nerds to use the new snowflake extension to debug the rendered code interactively  as captured from the output.

**Comment by MACKAT05** [comment]
Optionally this loop over command could be incorporated into an expanded render command syntax... as per #156

**Comment by sfc-gh-tmathew** [comment]
Interesting @MACKAT05. Using the Render to verify the sql that will be executed appears to be useful application of the `render` functionality. It is as if we need a `--dry-run` option for the render command without incrementing the version of the SQL files to during the development lifecycle.

How do you think this pattern will fare for CI/CD based deployment workflows in Development environment? Are you envisioning the iterative development is done locally, say on a clone and then pushed for actual deployment in Development and higher environments?

**Comment by sfc-gh-tmathew** [comment]
This appears to be a major refactor. Appears to be a 4.0 release candidate. Will get back to you @MACKAT05 once I have a chance to sync up with @jeremiahhansen.

**Comment by zanebclark** [comment]
@MACKAT05 , in  #275  I've split the cli.py file out into multiple files / directories to make it easier to write unit tests and such. I poked around your fork and couldn't find a branch with the folder structure above. Is that work in a local branch that you haven't committed? Do you think you could build your feature off of #275?

**Comment by MACKAT05** [comment]
yeah sorry about that it is still on my local... i'll try to get that visible and up later tonight.
